### PR TITLE
Add Pace integration to PostHog

### PR DESCRIPTION
**Pull request by neilkakkar**

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -550,5 +550,14 @@
     "maintainer": "official",
     "displayOnWebsiteLib": true,
     "type": "ingestion_filtering"
+  },
+  {
+    "name": "Pace Integration",
```
Comment (SaimonAlam15) [inline plugins.json:555]: Hey @neilkakkar, sorry for the last minute comment. The name of the app should be just 'Pace'. Should I push that change to my repo?

Comment (neilkakkar) [reply plugins.json:555]: Ah d'oh, my bad, no issues, the repository name makes sense, I'lljust change it here

Comment (neilkakkar) [inline plugins.json:555]:
```suggestion
    "name": "Pace",
```

File: plugins.json [hunk 2]
```diff
@@ -550,5 +550,14 @@
     "maintainer": "official",
     "displayOnWebsiteLib": true,
     "type": "ingestion_filtering"
+  },
+  {
+    "name": "Pace",
+    "url": "https://github.com/PostHog/pace-posthog-integration",
+    "description": "Send event data to Laudspeaker",
```
Comment (joethreepwood) [inline plugins.json:557]:
```suggestion
    "description": "Forward events directly to Pace",
```

**Comment by joethreepwood** [comment]
In case anyone wants to take a look at that too: https://github.com/PostHog/posthog.com/pull/4746

**Pull request merged**